chapters/scoping.tex

```diff
@@ -734,7 +722,7 @@ \subsection{Generation of the flat equation system}\label{generation-of-the-flat
     If it is of a non-simple type the instance is recursively handled.
   \end{itemize}
 \item
-  If there are equation or algorithm sections in the class definition of the instance, references are resolved using the instance scope of the instance and are included in the equation system.  Some references -- in particular to non simple, non record objects like connectors in connect statements and states in transition statements are not resolved yet and handled afterwards.
+  If there are equation or algorithm sections in the class definition of the instance, references are resolved using the instance scope of the instance and are included in the equation system.  Some references -- in particular to non simple, non record objects like connectors in connect-equations and states in \lstinline!transition! equations are not resolved yet and handled afterwards.
```

Comment:
I think connect should be treated similarly as transition here.

Reply:
I find 23 occurrences of "connect-equations", and 4 occurrences of "`connect` equations".  Wouldn't it be good to be consistent?  To be the logical resolution would be to change 4 ones, and say _transition-equations_.

(I'm not at all sure it's a good thing that I've become so used to the practice of using a hyphen to indicate that something isn't a ordinary language term, that this is how I suggest to resolve a situation like this.  The fact that it took me a long time to come to this position could also be an indication that a typical user would find it less disruptive to see "`connect` equation" compared to the half-baked language term "connect-equation".  Anyway, even if we would decide that "`connect` equation would be better in the end, I think we should strive only for consistency in this PR, and align with "connect-equation".  Changing all of them to "`connect` equation" would be easy to do as long as everything is consistent (and there aren't any unpredictable line breaks in the source code!).)

Reply:
Filed #2713 to settle this.  Enough to resolve?